dashboard/tests/run_ui_tests.sh

```diff
@@ -12,6 +12,7 @@ trap clean_up EXIT
 
 echo "Installing cypress"
 if [ -n "$BUILDKITE" ]; then
+  sudo apt-get update
```

Comment:
could you also change the next line of `apt install` into `apt-get install`?

and add sudo (or remove all sudos) for consistency?